helm/starboard-exporter/values.yaml

```diff
@@ -3,6 +3,7 @@ registry:
 image:
   name: "giantswarm/starboard-exporter"
   tag: "[[ .Version ]]"
+  pull_secret_enabled: true
```

Comment:
```suggestion
```
I think the user should have to opt-in here, not give them a blank secret by default

Reply:
I personally don't have any need for it. I don't know if you do ether, it was just a default config that I assumed you wanted in all your helm charts. If you don't see any need I will remove it all together.

Reply:
I pushed a commit where it's removed all together